addrmgr/addrmanager_test.go

```diff
@@ -105,10 +107,35 @@ func lookupFunc(host string) ([]net.IP, error) {
 	return nil, errors.New("not implemented")
 }
 
+func newAddrManager(t *testing.T, testName string, localSubnetworkID *subnetworkid.SubnetworkID) (*AddrManager, func()) {
+	dbPath, err := ioutil.TempDir("", testName)
+	if err != nil {
+		t.Fatalf("Error creating temporary directory: %s", err)
+	}
+
+	err = dbaccess.Open(dbPath)
+	if err != nil {
+		t.Fatalf("error creating db: %s", err)
+	}
+
+	amgr := New(lookupFunc, localSubnetworkID)
```

Comment:
I know that AddressManager is full of terrible names, but let's no proliferate that even more, and use only full words in the new names we create